tests/cases/fourslash/indentationInAmdIife.ts

```diff
@@ -0,0 +1,51 @@
+/// <reference path="fourslash.ts"/>
+
+//// function foo(a?,b?) { b(a); }
+//// 
+//// (foo)(1, function() {/*4_0*/
+//// });
+//// 
+//// // No line-breaks in the expression part of the call expression
+//// 
+//// (foo)
+////     (1, function () {/*8_0*/
+////     });
+//// (foo)(1,
+////     function () {/*8_1*/
+////     });
+//// (foo)(1, function() 
+//// {/*4_2*/
+//// });
+//// 
+//// // Contains line-breaks in the expression part of the call expression.
+////
+//// (
+////     foo)(1, function () {/*4_1*/

```

Comment:
This is a consequence of the algorithm. We can fix this later if it crops up, but it's tricky to change without special-casing because we are simulating the argument-list as a parent of the lambda.